support/test/cljsbuild/test/compiler.clj

```diff
@@ -55,15 +65,23 @@
     (util/find-files cljs-path-b #{"clj"}) => [] :times 1
     (util/find-files cljs-path-a #{"cljs"}) => [cljs-file-a] :times 1
     (util/find-files cljs-path-b #{"cljs"}) => [cljs-file-b] :times 1
+    (util/find-files cljs-checkout-path-a #{"clj"}) => [] :times 1
+    (util/find-files cljs-checkout-path-b #{"clj"}) => [] :times 1
+    (util/find-files cljs-checkout-path-a #{"cljs"}) => [cljs-checkout-file-a] :times 1
+    (util/find-files cljs-checkout-path-b #{"cljs"}) => [cljs-checkout-file-b] :times 1
```

Comment:
I track cljc file if your environment allow it. cljc support is enabled by condition (https://github.com/emezeske/lein-cljsbuild/blob/master/support/src/cljsbuild/compiler.clj#L119). *support* project depends on clojure 1.5 and clojurescript 0.0-2411 and cljc files are not tracked in unit tests.